trollflow2/tests/test_trollflow2.py

```diff
@@ -2010,5 +2010,30 @@ def fake_persist(*args):
     assert not sc_3a_3b["another"].attrs.get("persisted")
 
 
+def test_format_decoration():
+    """Test that decoration text in fmt_config is formated based on fmat."""
+    import datetime
+    from trollflow2.plugins import format_decoration
+    # set input data
+    fmat = {'orig_platform_name': 'npp',
+            'start_time': datetime.datetime(2022, 5, 3, 12, 7, 52)}
+    fmat_config = {'format': 'png', 'writer': 'simple_image',
+                   'decorate': {'decorate': [{'text': {'txt': '{start_time:%Y-%m-%d %H:%M}',
+                                                       'align': {'top_bottom': 'top', 'left_right': 'right'}}}]}}
+    # set ground truth
+    formated = {'format': 'png', 'writer': 'simple_image',
+                'decorate': {'decorate': [{'text': {'txt': '2022-05-03 12:07',
+                                                    'align': {'top_bottom': 'top', 'left_right': 'right'}}}]}}
+    # test valid input
+    assert format_decoration(fmat, fmat_config) == formated
+
+    # set invalid input data
+    fmat_config_invalid = {'format': 'png', 'writer': 'simple_image',
+                           'decorate': {'decorate': [{'text': {'txt': '{time:%Y-%m-%d %H:%M}',
+                                                      'align': {'top_bottom': 'top', 'left_right': 'right'}}}]}}
+    # test invalid input
+    assert format_decoration(fmat, fmat_config_invalid) == fmat_config_invalid
```

Comment:
This should be a separate test case.

Am I right in assuming that the functionality is basically such, that if formatting doesn't work, the "invalid" text will be used instead?